deploy/kubernetes/overlays/prow-gke-release-staging-rc/sidecar-upgrade.yaml

```diff
@@ -0,0 +1,17 @@
+- op: remove
+  path: /spec/template/spec/containers/0/args/5
+
+- op: remove
```

Comment:
can you put a comment on the arg being removed?

Reply:
Done